src/pages/docs/getting-started/laravel.mdx

```diff
@@ -0,0 +1,398 @@
+
+                if (!message.trim()) {
+                    alert('Please enter a message');
+                    return;
+                }
+
+                // Send message to Laravel API
+                fetch('/api/public-event', {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        'X-CSRF-TOKEN': csrfToken,
+                        'Accept': 'application/json'
+                    },
+                    body: JSON.stringify({
+                        channelName: channelName,
+                        message: message
+                    })
+                })
+                .then(response => response.json())
+                .then(data => {
+                    console.log('Message sent successfully:', data);
+                    document.getElementById('messageInput').value = ''; // Clear the input
+                    document.getElementById('messages').innerHTML +=
+                        `<div class="message py-2 border-b border-gray-100 text-gray-600"><strong>Sent:</strong> ${message} (to ${channelName})</div>`;
+                })
+                .catch(error => {
+                    console.error('Error sending message:', error);
+                    alert('Error sending message. Check console for details.');
+                });
+            });
+        });
+    </script>
+</body>
+</html>
+```
+</Code>
+
+Add the route to serve the frontend file in `routes/web.php` to expose this new file to the browser:
+
+<Code>
+```php
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+Route::get('/', function () {
+    return view('welcome');
+});
+
+Route::get('/echo', function () {
+    return view('echo');
+});
+```
+</Code>
+
+In your browser, open the url `http://localhost:8000/echo` to see the frontend page.
+
+Use `curl` or any HTTP client to send a POST request, you should see the message appear in the `#messages` div on the page:
+
+<Code>
+```shell
+curl --location --request POST 'localhost:8000/api/public-event' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "channelName":"my-first-channel",
+    "message":"A message sent from my first client!"
+}'
+```
+</Code>
+
+![Screenshot showing the Laravel Echo with Ably test page running in a browser](../../../images/content/screenshots/getting-started/laravel-echo-example.png)
+
+## Next steps <a id="next-steps"/>
+
+Continue to explore the documentation with Laravel as the selected language:
+
+Read more about the concepts covered in this guide:
+
+* Read more about [Ably Laravel Echo](https://github.com/ably-forks/laravel-echo)
+* Fetch message [history](/docs/storage-history/history?lang=php) in your apps
+* Learn more about [Laravel Broadcasting](https://laravel.com/docs/broadcasting)
+* Explore the A[bly Laravel Broadcast app](https://github.com/ably-labs/laravel-broadcast-app), with authentication, multiple rooms, and private rooms.
```

Comment:
```suggestion
* Explore the [Ably Laravel Broadcast app](https://github.com/ably-labs/laravel-broadcast-app), with authentication, multiple rooms, and private rooms.
```

Reply:
I think we should add some context around what this app is if we're linking out to it.